views/director-residential-address-search-choose-address.html

```diff
@@ -39,6 +38,13 @@ <h1 class="govuk-heading-l">Choose an address</h1>
           id: "address_array",
           name: "address_array",
           value: address_array,
+          fieldset: {
+            legend: {
+              text: "Choose an address",
```

Comment:
I see I had it wrong initially but should it be "Choose a home address"?
It was in the ACs of the skeleton page but missed?

Reply:
Nope the AC1 [here](https://companieshouse.atlassian.net/browse/DACT-802) says it should be 'Choose an address'